src/quick-start.md

```diff
@@ -28,13 +28,30 @@ helm repo add kubewarden https://charts.kubewarden.io
 helm install --wait -n kubewarden --create-namespace kubewarden-crds kubewarden/kubewarden-crds
 
 helm install --wait -n kubewarden kubewarden-controller kubewarden/kubewarden-controller
+
+helm install --wait -n kubewarden kubewarden-defaults kubewarden/kubewarden-defaults
 ```
 
 The following charts should be installed inside the `kubewarden` namespace in your Kubernetes cluster:
 
 * `kubewarden-crds`, which will register the `ClusterAdmissionPolicy` and `PolicyServer` Custom Resource Definitions
 
-* `kubewarden-controller` with a default configuration, and which will create a `PolicyServer` resource named `default`. 
+* `kubewarden-controller`, which will install the Kubewarden controller
+
+* `kubewarden-defaults`, which will create a `PolicyServer` resource named `default`.
```

Comment:
This can also install a set of recommended policies to secure your cluster by enforcing some well known best practices.